google-cloud-storage/src/main/java/com/google/cloud/storage/multipartupload/model/ListPartsResponse.java

```diff
@@ -0,0 +1,365 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.storage.multipartupload.model;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.google.api.core.BetaApi;
+import com.google.cloud.storage.StorageClass;
+import com.google.common.base.MoreObjects;
+import java.util.List;
+import java.util.Objects;
+
+/** Represents a response to a list parts request. */
+@BetaApi
+public final class ListPartsResponse {
+
+  @JacksonXmlProperty(localName = "Bucket")
+  private String bucket;
+
+  @JacksonXmlProperty(localName = "Key")
+  private String key;
+
+  @JacksonXmlProperty(localName = "UploadId")
+  private String uploadId;
+
+  @JacksonXmlProperty(localName = "PartNumberMarker")
+  private int partNumberMarker;
+
+  @JacksonXmlProperty(localName = "NextPartNumberMarker")
+  private int nextPartNumberMarker;
+
+  @JacksonXmlProperty(localName = "MaxParts")
+  private int maxParts;
+
+  @JsonAlias("truncated") // S3 returns "truncated", GCS returns "IsTruncated"
+  @JacksonXmlProperty(localName = "IsTruncated")
+  private boolean isTruncated;
+
+  @JacksonXmlProperty(localName = "Owner")
+  private Owner owner;
```

Comment:
Owner isn't listed in the public docs? is that something we should get added to the docs? https://cloud.google.com/storage/docs/xml-api/get-bucket-uploads#response_body_elements